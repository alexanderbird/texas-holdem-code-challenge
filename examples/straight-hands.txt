AH QH JH 9C 8H
Anne-marie KH TH
Damaris TD 8D
Gerik TC 8S
Melicent 9H TH
Jaimie 8C 7C 6C 5C
