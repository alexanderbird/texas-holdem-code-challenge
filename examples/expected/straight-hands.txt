1 Anne-marie Royal Flush
2 Melicent Straight Flush Queen
3 Jaimie Straight Flush 9
5 Damaris Straight Queen
5 Gerik Straight Queen
